src/test/java/unit/CloneWorkspace.java

```diff
@@ -167,7 +167,7 @@ public void cloneWorkspace(TestInfo testInfo) throws Exception {
 
     // Update workspace name. This is for testing PF-1623.
     TestCommand.runAndParseCommandExpectSuccess(
-        UFWorkspace.class, "workspace", "update", "--name=update_Name");
+        UFWorkspace.class, "workspace", "update", "--new-name=update_Name");
```

Comment:
I know this was before your change, but `update_Name` looks weird. Let's change to `update_name`.